doc/flux_to_redux.md

```diff
@@ -0,0 +1,579 @@
+
+    The result of these rules is that a store can be converted directly to Redux [iff](https://en.wikipedia.org/wiki/If_and_only_if) it will be used exclusively by Redux components, but a component can be converted to a Redux component if it will pull data from either an adapted Influx store or a traditional Flux store.
+
+    If the stores are not overly complex and all of the related UI can be updated to Redux connected components (extending from `UiComponent2`), then going straight to Redux may be a good option.
+
+    If, given the size of the store or number of components affected, this is not ideal, the store should be converted to Influx. 
+1. __Refactor the store.__ 
+    If the store is moving to Redux, follow the [simple store conversion](#basic-conversion-step-by-step). If the store is moving to Influx:
+    1. __Create the Redux actions.__ Any actions that the connected UI components will need access to should be made into a Redux action.
+    1. __Create a Redux state class for the store.__ Any state that is mutated by the Redux actions should be present in the Redux state class.
+    1. __Create the Redux reducer for the store.__ This should just combine the actions and the Redux state class - any action should have a condition within the reducer, and ultimately all fields on the state store should have a way to be updated.
+    1. __Convert the Flux store to an Influx store:__
+        1. __Add the `InfluxStoreMixin` to the original Flux store.__ This is what makes it an "Influx" store, and adds the fields required by an Influx architecture. The model for `state` field is the Redux state class created earlier, and it should be passed into the mixin's typing.
+        1. __Initialize the default Redux state in the Influx contructor.__ 
+        1. __Refactor actions passed into `triggerOnActionV2`.__ Like a Flux store, the Influx store will watch for Flux actions to be triggered. Instead of triggering a function that mutates the inner state however, a callback should pass a corresponding Redux action instance into `this.influxReducer`.
+        1. __Override the `reduxReducer` getter on the class.__ The `reduxReducer` getter should point to the Redux reducer function created earlier.
+        1. __Update the store fields.__ The end result of this step is that the Influx store (previously the Flux store) should have a getter for every Redux state class field. The getters should point to corresponding values on the `state` field. 
+        
+        Updating getters isn't necessary as the correct fields should be accessible via `influxStoreInstance.state.reduxProperty`, but refactoring the getters means that any Flux components pointing to this store do not need to be refactored. If it is preferable just to remove getters in favor of accessing the `state` fields directly, any original getters or fields that correspond to a value moved to the Redux state class should be removed.
+
+        ```dart
+        // An Influx Store
```

Comment:
I think we should show the necessary imports for clarity as to what the `flux` namespace represents, and where they can access stuff like `InfluxStoreMixin`.

```suggestion
        import 'package:over_react/over_react_redux.dart';
        import 'package:w_flux/w_flux.dart' as flux;

        // An Influx Store
```